documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1609,100 +1609,35 @@ If the `continuous` parameter is set to `1`, the job pauses for the length of ti
-
-. Use link:https://access.redhat.com/products/red-hat-amq#streams[Red Hat AMQ Streams] to set up Apache Kafka and Kafka Connect on OpenShift. AMQ Streams offers operators and images that bring Kafka to OpenShift.
-
-. From a browser, open the link:https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=red.hat.integration&downloadType=distributions[Software Downloads page on the Red Hat Customer Portal],
-and download the {prodname} SQL Server connector.
-
-. Extract the connector files into your Kafka Connect environment.
-. Add the connector plug-in's parent directory to your Kafka Connect `plugin.path`, for example:
-+
-[source]
-----
-plugin.path=/kafka/connect
-----
-+
-The preceding example assumes that you extracted the {prodname} SQL Server connector to the `/kafka/connect/{prodname}-connector-sqlserver` path.
-
-. Restart your Kafka Connect process to ensure that it picks up the new JAR files.
-
-You also need to {LinkDebeziumUserGuide}#setting-up-sqlserver[set up SQL Server] to run a {prodname} connector.
-
-.Additional resources
-
-For more information about the deployment process, and deploying connectors with AMQ Streams, see the {prodname} installation guides.
+To deploy a {prodname} SQL Server connector, you install the {prodname} SQL Server connector archive, configure the connector, and start the connector by adding its configuration to Kafka Connect.
 
-* {LinkDebeziumInstallOpenShift}[{NameDebeziumInstallOpenShift}]
-* {LinkDebeziumInstallRHEL}[{NameDebeziumInstallRHEL}]
+.Prerequisites
+* link:https://zookeeper.apache.org/[Zookeeper], link:http://kafka.apache.org/[Kafka], and link:{link-kafka-docs}.html#connect[Kafka Connect] are installed.
+* SQL Server is installed and is {link-prefix}:{link-sqlserver-connector}#setting-up-sqlserver[set up to run the {prodname} connector].
 
-endif::product[]
+.Procedure
+. Download the {prodname} https://repo1.maven.org/maven2/io/debezium/debezium-connector-sqlserver/{debezium-version}/debezium-connector-sqlserver-{debezium-version}-plugin.tar.gz[SQL Server connector plug-in archive]
+. Extract the files into your Kafka Connect environment.
+. Add the directory with the JAR files to {link-kafka-docs}/#connectconfigs[Kafka Connect's `plugin.path`].
+. {link-prefix}:{link-sqlserver-connector}#sqlserver-example-configuration[Configure the connector] and {link-prefix}:{link-sqlserver-connector}#sqlserver-adding-connector-configuration[add the configuration to your Kafka Connect cluster.]
+. Restart your Kafka Connect process to pick up the new JAR files.
 
+If you are working with immutable containers, see link:https://hub.docker.com/r/debezium/[{prodname}'s container images] for Zookeeper, Kafka, SQL Server and Kafka Connect with the SQL Server connector already installed and ready to run. You can also xref:operations/openshift.adoc[run {prodname} on Kubernetes and OpenShift].
```

Comment:
Debezium doesn't provide a container image for SQL Server. Instead, Microsoft's official image should be used and referenced here: https://hub.docker.com/_/microsoft-mssql-server.

Reply:
Also "Apache Kafka" and "Apache ZooKeeper" (upper-case K).

Reply:
> Debezium doesn't provide a container image for SQL Server. Instead, Microsoft's official image should be used and referenced here: https://hub.docker.com/_/microsoft-mssql-server.

Ah, ok. I'll make that update. The statement about immutable containers appears in the current 1.4/1.5 upstream doc.

Reply:
Yes, that's good. Only use "... for Apache ZooKeeper, Apache Kafka, and Kafka Connect.